pkg/app/resource_event_handler.go

```diff
@@ -38,34 +43,46 @@ func (h *resourceEventHandler) OnUpdate(oldObj, newObj interface{}) {
 }
 
 func (h *resourceEventHandler) OnDelete(obj interface{}) {
-	meta, ok := obj.(meta_v1.Object)
+	metaObj, ok := obj.(meta_v1.Object)
 	if !ok {
 		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
 		if !ok {
 			log.Printf("[REH] Delete event with unrecognized object type: %T", obj)
 			return
 		}
-		meta, ok = tombstone.Obj.(meta_v1.Object)
+		metaObj, ok = tombstone.Obj.(meta_v1.Object)
 		if !ok {
 			log.Printf("[REH] Delete tombstone with unrecognized object type: %T", tombstone.Obj)
 			return
 		}
 	}
-	bundleName, namespace := getBundleNameAndNamespace(meta)
-	h.rebuildByName(namespace, bundleName, "deleted", obj)
```

Comment:
This line was using `obj` but it should be `meta`/`metaObj` because it is the variable that is updated when tombstones are handled (line 48/53).